Tests/dep/Utilities.swift

```diff
@@ -55,11 +55,11 @@ func fixture(name fixtureName: String, tag: String = "1.2.3", @noescape body: (S
 }
 
 
-func executeSwiftBuild(chdir: String) throws {
+func executeSwiftBuild(chdir: String, redirectStandardError: Bool = false) throws -> String {
     let toolPath = Resources.findExecutable("swift-build")
     var env = [String:String]()
     env["SWIFT_BUILD_TOOL"] = getenv("SWIFT_BUILD_TOOL")
-    try system([toolPath, "--chdir", chdir], environment: env)
+    return try popen([toolPath, "--chdir", chdir], redirectStandardError: redirectStandardError, printOutput: true, environment: env)
```

Comment:
I took some pains to try to make sure that the output from `system` was equivalent to the output from `popen` with the `printOutput` flag enabled, so I'm hoping that it add noise (with the exception of the additional log message that I added)